src/components/PathwayExercise/index.js

```diff
@@ -246,19 +259,25 @@ function PathwayExercise() {
                 </Link>
               </Typography>
 
-              <Select
-                disableUnderline
-                value={language}
-                IconComponent={() => null}
-                onChange={(e) => {
-                  setLanguage(e.target.value);
-                }}
-                variant="standard"
-              >
-                <MenuItem value="en">English</MenuItem>
-                <MenuItem value="hi">Hindi</MenuItem>
-                <MenuItem value="mr">Marathi</MenuItem>
-              </Select>
+              {availableLang?.length === 1 ? (
+                <Select
+                  disableUnderline
+                  value={language}
+                  IconComponent={() => null}
+                  onChange={(e) => {
+                    setLanguage(e.target.value);
+                  }}
+                  variant="standard"
+                >
+                  {availableLang.map((lang) => {
+                    return <MenuItem value={lang}>{Lang[lang]}</MenuItem>;
+                  })}
+                </Select>
+              ) : (
+                <MenuItem value={availableLang[0]}>
+                  {Lang[availableLang[0]]}
+                </MenuItem>
+              )}
```

Comment:
Was this meant to be reversed (`<Select>` for when there are multiple available languages) to match the above?  If so, you should create a new Component to avoid code duplication, which as illustrated here can easily lead to inadvertent inconsistencies.